internal/controller/standalone_pgadmin/pod.go

```diff
@@ -325,10 +311,15 @@ loadServerCommand
 	// descriptor and uses the timeout of the builtin `read` to wait. That same
 	// descriptor gets closed and reopened to use the builtin `[ -nt` to check mtimes.
 	// - https://unix.stackexchange.com/a/407383
+	// In order to get gunicorn to reload the logging config
+	// we need to send a KILL rather than a HUP signal.
+	// - https://github.com/benoitc/gunicorn/issues/3353
+	// Right now the config file is on the same configMap as the cluster file
+	// so if the mtime changes for any of those files, it will change for all.
 	var reloadScript = `
 exec {fd}<> <(:||:)
 while read -r -t 5 -u "${fd}" ||:; do
-    if [[ "${cluster_file}" -nt "/proc/self/fd/${fd}" ]] && loadServerCommand
+    if [[ "${cluster_file}" -nt "/proc/self/fd/${fd}" ]] && loadServerCommand && kill -KILL $(head -1 ${PGADMIN4_PIDFILE?});
```

Comment:
🌱 I expect this increases the restart count when changing settings. WFM!

Reply:
I haven't seen any restarts with KILL -- our gunicorn is wrapped by our loop script.